### DISCUSSION TITLE
Line breaks being lost on insertText command

### DISCUSSION THREAD

**Discussion by viveleroi** [tinymce/tinymce#9902] [General]
I'm trying to insert text with newlines and it's failing, the text becomes inserted as a single line. I assume tiny would handle converting newlines to `p` or `br` tags?

```
editor.execCommand(
      'insertText',
      false,
      `foo
bar
three`
    )
```

**Comment by TheSpyder** [comment]
If you're trying to replicate what happens when plain text is pasted, you're looking for `mceInsertClipboardContent`.
https://www.tiny.cloud/docs/tinymce/latest/editor-command-identifiers/#miscellaneous-core-commands
